### ISSUE TITLE
Sweeping change to make button height accessible 

### ISSUE THREAD

**Issue by anitadesigns** [hackforla/expunge-assist#689]
### Overview
The design team would like to make our website more accessible. WCAG 2.1 shares that button heights on mobile should be 44px. When looking back at our own buttons, they fall about 4px short. We would like to update the button height from 40px to 44px on the mobile breakpoint. 

### Action Items
Pages that have a button:
- [ ] [landing page](https://expungeassist.org/#/)
- [ ] [ About Us page](https://expungeassist.org/#/./about-us)
- [ ] https://expungeassist.org/#/./welcome
- [ ]  https://expungeassist.org/#/./before-you-begin
- [ ] https://expungeassist.org/#/./advice
- [ ] https://expungeassist.org/#/./form/intro
- [ ] https://expungeassist.org/#/./form/introduction/preview
- [ ] https://expungeassist.org/#/./form/involvement
- [ ] https://expungeassist.org/#/./form/job
- [ ] https://expungeassist.org/#/./form/job/preview
- [ ] https://expungeassist.org/#/./form/recovery
- [ ] https://expungeassist.org/#/./form/recovery/preview
- [ ] https://expungeassist.org/#/./form/school
- [ ] https://expungeassist.org/#/./form/school/preview
- [ ] https://expungeassist.org/#/./form/parenting
- [ ] https://expungeassist.org/#/./form/parenting/preview
- [ ] https://expungeassist.org/#/./form/community-service
- [ ] https://expungeassist.org/#/./form/community-service/preview
- [ ] https://expungeassist.org/#/./form/unemployed
- [ ] https://expungeassist.org/#/./form/unemployed/preview
- [ ] https://expungeassist.org/#/./form/goals
- [ ] https://expungeassist.org/#/./form/goals/preview
- [ ] https://expungeassist.org/#/./form/why
- [ ] https://expungeassist.org/#/./form/why/preview
- [ ] https://expungeassist.org/#/./form/finalize
- [ ] https://expungeassist.org/#/./form/finalize/preview

### Resources/Instructions
Find design details in Figma under the "Design handoff page"

**Comment by aahvocado** [comment]
Moving this task to be done in #688
